broker/src/main/java/org/apache/rocketmq/broker/processor/PullMessageProcessor.java

```diff
@@ -87,8 +87,8 @@ private RemotingCommand processRequest(final Channel channel, RemotingCommand re
 
         response.setOpaque(request.getOpaque());
 
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("receive PullMessage request command, {}", request);
+        if (log.isDebugEnabled()) {
```

Comment:
`log.isDebugEnabled()` is not necessary, and do not consistent with other codes log-style.